### PR TITLE
Added Documentation to Install Kubeflow 1.5 on Azure

### PR DESCRIPTION
**Pull request by ajinkya933**
Documentation added to Kubeflow website which shows how to install Kubeflow 1.5 on Azure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by ajinkya933** [comment]
please test

File: content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+1. Check installed version:
+
+    ```
+    kubectl version --client
+    ```
+
+1. Run the following commands to set up and deploy Kubeflow. 
+    ```
+    git clone https://github.com/kubeflow/manifests
```
Comment (yhwang) [inline content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md:91]: I guess this is for kubeflow v1.5, do you want to pin the branch here? for kubeflow 1.5, there is `v1.5-branch` branch.

Comment (ajinkya933) [reply content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md:91]: It is for Kubeflow 1.5 how do I pin the branch here ?

**Comment by thesuperzapper** [comment]
@yhwang @ajinkya933 what is the status on this PR?

File: content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md (hunk 1 above)
Comment (ajinkya933) [reply content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md:91]: @yhwang I checked, this PR is a website-documentation update and it should not be dependent on a branch, as its pushed to Kubeflow Website.

**Comment by ajinkya933** [comment]
@thesuperzapper Its a Kubeflow website update which needs approval

**Comment by yhwang** [comment]
@thesuperzapper 
> what is the status on this PR?

I don't have AWS to try it out, but I am good with the content. just one comment about the branch of manifests repo. but it's up to @ajinkya933 to structure his doc for different kubeflow versions.

File: content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md (hunk 1 above)
Comment (ajinkya933) [reply content/en/docs/distributions/azure/deploy/install-kubeflow-1_5.md:91]: @yhwang I have created PR for branch 1.5 here: https://github.com/kubeflow/website/pull/3284 please review

**Comment by ajinkya933** [comment]
I have created PR for 1.5 branch here: https://github.com/kubeflow/website/pull/3284

**Pull request closed without merging**